### ISSUE TITLE
Update Aztec Starter to use Boxes

### ISSUE THREAD

**Issue by critesjosh** [AztecProtocol/dev-rel#165]
Aztec Starter is currently an independent [repo](https://github.com/AztecProtocol/aztec-starter/tree/main). Once @signorecello's Boxes updates are complete, update the starter to be a clone of the simple box.

**Comment by critesjosh** [comment]
blocked by #145

**Comment by signorecello** [comment]
Just an update on this. As discussed in person, I still secretly think Aztec-Starter is the way to go. 

Boxes can be massive because they'll provide us with a starter that is 100% in sync with the code and is not owned by DevRel. But we need to finish #145 in its entirety before re-evaluation IMO

**Comment by signorecello** [comment]
Are we ready now @critesjosh ? I think we should wait for #179 (if we decide to do it) before pulling the trigger on aztec-starter, for feature parity

**Comment by critesjosh** [comment]
After discussing with @signorecello, we plan to deprecate aztec-starter in favor of running boxes directly in codespaces from the monorepo. Developers should be able to clone the box into a new repo from the running codespace.

**Comment by critesjosh** [comment]
closing in favor of https://github.com/AztecProtocol/dev-rel/issues/222

**Comment by signorecello** [comment]
Just enabled auto merge on [this little friend ](https://github.com/AztecProtocol/aztec-packages/pull/5752), I'll just double check it all goes well in master and proceed
